f8a_tagger/tokenizer.py

```diff
@@ -34,19 +36,27 @@ def __init__(self, stopwords_file=None, ngram_size=1, lemmatizer=None, stemmer=N
         self._stemmer = stemmer
         self._lemmatizer = lemmatizer
 
-        with open(stopwords_file or self._STOPWORDS_TXT, 'r') as f:
-            for word in f.read().split('\n'):
-                # Remove comments
-                if word.startswith('#') and word != '#':
-                    continue
+        if isinstance(stopwords_file, str) or stopwords_file is None:
```

Comment:
I agree with this. Sorry, my earlier comment is more than actual, infact it may show errors in case of stopword_files as None and we don't have our  self._STOPWORDS_TXT. 
Otherwise, every each piece of code looks fine and can be merged.